### PR TITLE
Add instructor portrait to the Cards

### PR DESCRIPTION
**Pull request by ltclmbdev**
giphy

### CODE REVIEW

File: src/components/Card/index.js [hunk 1]
```diff
@@ -103,7 +105,10 @@ const CardBody = ({title, instructor, type}) => {
   return (
     <div className={`${type === 'playlist' ? 'ph4' : ''}`}>
       <h3 className={titleHeadingClasses}>{title}</h3>
-      <div className={instructorNameClasses}>{instructor}</div>
+      <div className='mb4 flex justify-center items-center'>
+        <img src={instructorPortrait} alt='' className='w2 h2 br-pill mr3 pointer'/>
```
Comment (ltclmbdev) [inline src/components/Card/index.js:109]: Should I put the `instructorPortrait` classes to something like `instructorPortraitClasses`?

Comment (trevordmiller) [reply src/components/Card/index.js:109]: I think they are fine inline with `className` as you have done :) IMO, it only makes sense to pull them out if they need to be shared across elements or transformed in some way.

**Review by trevordmiller — APPROVED** [review]
Looks good @DrShpongle. Please wait to merge until after I get the new egghead-ui deployment working though. I'll post an update in `#egghead-workin-on` when it is working.

File: src/components/Card/index.js (hunk 1 above)
Comment (ltclmbdev) [reply src/components/Card/index.js:109]: That is why I was in doubt, thanks Trevor!

**Pull request merged**